components/brave_sync/ui/containers/enabledContent.tsx

```diff
@@ -70,6 +70,15 @@ export default class SyncEnabledContent extends React.PureComponent<Props, State
     }
   }
 
+  componentDidMount () {
+    const { syncData } = this.props
+    // if only one device is found, remind the user
+    // at every page refresh to add a new device
+    if (syncData.devices.length < 2) {
+      this.setState({ viewSyncCode: true })
```

Comment:
actually, showing `device type` would be more useful because user might not only add desktop as second device
<img width="698" alt="Screen Shot 2019-12-30 at 15 37 27" src="https://user-images.githubusercontent.com/11330831/71604964-61e89d00-2b1a-11ea-84fd-137d1dfe4994.png">

Reply:
done